docs/markdown/how-to.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to work with us
+description: Guidelines on how to engage with the Gestalt team, when to work with us, and how to contribute.
+---
+
+## What goes into Gestalt?
+
+The goal of Gestalt is to create a shared library of design best practices, React components, and documentation. Therefore, the best contenders for addition to Gestalt are designs or components that benefit multiple teams and are used regularly throughout our products.
+
+## Requesting new components or changes
+
+**Please note:** The process below applies to Pinterest employees.
+We ask that any ideas that change the functionality of an existing Gestalt component or introduce a net-new component go through the following process:
+
+1. **Present work early in our [partnership meetings](#Meetings-and-events)**
+   What we build into Gestalt comes from the teams across Pinterest, so we’d love to pair with you early and often! Seeing early mocks, wireframes, or even product roadmaps and learning where your project may be going helps inform our team OKRs. The earlier we see the work, the better we're able to plan and the higher the likelihood we'll be able to help. Please [sign up](https://pinch.pinadmin.com/gestaltSignUp) for an Office Hours slot with your discussion topic.
+2. **Iterate on solutions**
```

Comment:
could we space this list items to make it more readable?

Reply:
Got it. I'll add the spacing. 

Btw, just wanted to call something out here that I noticed. It seems that depending on the Markdown renderer used, lists with text following it are rendered differently. A little subtle.

For example, Github Renders the following as
1. **Bold Header**
Text underneath

In the [MDX renderer](https://mdxjs.com/playground/), it handles the same case to show up as.
1. **Bold Header** Text shows up here

Different [renderers](https://markdownlivepreview.com/) seem to handle in their own way.

The text continues on the same line after a bulleted item. A line break isn't inserted by default in the MDX markdown. This is most likely because a bold item is technically an inline item, and the renderer doesn't insert a break right after. 

A few ways to [fix this](https://gist.github.com/shaunlebron/746476e6e7a4d698b373):
1. Add two spaces after the bulleted header
2. Use an explicit `<br/>` or a `\` at the end of the line (which I did)
3. Use a Markdown Header instead (e.g #### H4). Ideal route. We'll have to figure out the margins here

I don't forsee it being a dealbreaker, thinking we can write around it, or tweak our writing to use headers.